shukla/Dockerfile

```
@@ -0,0 +1,12 @@
+FROM ubuntu:22.04
+
+COPY helloWorld.java .
```

```suggestion
COPY helloWorld.java .
COPY Makefile .
```

RE: https://github.com/AFIDSI/student-developer-setup/pull/8#issuecomment-1475646582 the `COPY` command here is only copying the `.java` file into the `WORKDIR` in the container image and nothing else. So inside the built image there is no `Makefile` and so `make` has nothing to operate on and so fails.